### DISCUSSION TITLE
Visibility of checkboxes in table

### DISCUSSION THREAD

**Discussion by psinger** [h2oai/wave#705] [Q&A] [answered]
If I set `multiple=True` in a table, I would like to make the checkboxes permanently visible, not only when I hover over them. Is this possible?

![image](https://user-images.githubusercontent.com/1677826/112805010-5b2a3c00-9075-11eb-9407-15c5d17979ec.png)

**Comment by lo5** [comment, marked as answer]
Should be possible. Tracked here: https://github.com/h2oai/wave/issues/713

**Comment by ryanchesler** [comment]
adding +1 for this. So far users on my app have been confused my table has checkbox functionality or if their selections are checked or not
